services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
```

Comment:
Use get_user_model instead of using User explicitly because the purpose of AUTH_USER_MODEL is to not hardcode main model